### PR TITLE
Add new jenkins job for horizon integration tests.

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

**Comment by aplanas** [comment]
LGTM: +1

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml [hunk 1]
```diff
@@ -39,6 +39,11 @@
           description: Optional, specify cct tests to run
 
       - string:
+          name: integration
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-scenario-1a.yaml]: IMO bad choice of a parameter name, `integration_tests` would be better. 
As the PR description is missing, not sure how this change is meant; should it cover also other integration tests for openstack or it's horizon specific? If it's horizon specific, then `horizon_tests` would be sufficient.

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+- job:
+    name: cloud-mkphyscloud-qa-test-horizon-ui-integration
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      Runs tempest smoke tests
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml]: copy&paste err

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      - string:
+          name: scenario_job_name
+          description: Optional; name of the scenario jenkins job that is used to trigger this job
+
+      - string:
+          name: scenario_build_number
+          description: Optional; scenario build number that triggered this job
+
+      - bool:
+          name: integrationtests
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml]: Is this parameter needed? This job only triggers the horizon integration tests and nothing else, so no further confirmation for running the test should be required.

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
+- job:
+    name: cloud-mkphyscloud-qa-test-horizon-ui-integration
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml]: As horizon is the UI for openstack, perhaps, the `-ui-` is a bit redundant in the name; something like `horizon_tests` should be ok, and putting more details in the description would provide more information on it.

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      Useful only for cloud that was built by mkcloud.
+      Mandatory parameter: hw_number
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    wrappers:
+      - build-name:
+          name: '#${BUILD_NUMBER} - ${scenario_name} - qa$hw_number - horizon-ui'
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml]: `horizon` instead `horizon-ui`, the shorter the build name the better it's readable in the jenkins UI.

File: scripts/jenkins/qa/cloud-mkphyscloud-qa-tests-trigger.yaml [hunk 6]
```diff
@@ -107,3 +112,22 @@
...
+          condition-expression: ${ENV,var="horizon-tests"}
+          steps:
+            - trigger-builds:
+                - project: cloud-mkphyscloud-qa-test-horizon-tests
+                  condition: ALWAYS
+                  block: true
+                  block-thresholds:
+                    build-step-failure-threshold: never
+                    failure-threshold: never
+                    unstable-threshold: never
```
Comment (vmoravec) [inline scripts/jenkins/qa/cloud-mkphyscloud-qa-tests-trigger.yaml]: All the thresholds above should rather be set to `FAILURE` as once the job is triggered requiring horizon tests to be run (boolean  `horizon-tests` set to true), we want the build to be shown as failed if those horizon tests fail.

**Pull request merged**